test/uri_benchmark.jl

```diff
@@ -1,45 +1,45 @@
+using HTTP
 using Unitful
 
-using HTTP
+include("resources/urlparser.jl")
```

Comment:
this file wasn't really ever meant to be tests, so maybe just remove it, or move it to a top level `benchmark/` directory